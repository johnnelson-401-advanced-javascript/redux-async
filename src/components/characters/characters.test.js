import React from 'react';
import { shallow } from 'enzyme';
import Characters from './characters';

describe('Characters list component', () => {
  it('renders a list of characters', () => {
    const characters = [{
      _id: '123',
      name: 'name',
      photoUrl: 'nope'
    }];
    const wrapper = shallow(<Characters characters={characters} />);
    expect(wrapper).toMatchSnapshot();
  });
});
