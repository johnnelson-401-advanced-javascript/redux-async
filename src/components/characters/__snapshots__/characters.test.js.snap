// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Characters list component renders a list of characters 1`] = `
<ul
  className="Characters"
>
  <li
    key="123"
  >
    <Character
      key="123"
      name="name"
      photoUrl="nope"
    />
  </li>
</ul>
`;
